app/src/scenes/Items/index.tsx

```diff
@@ -35,32 +31,8 @@ class Items extends React.Component<
 
   componentDidMount() {
     this.fetchItems();
-    this.fetchSubCategoryDetails();
```

Comment:
You have to load it when the component gets mounted.

Reply:
We use this to get the subcategory title but now we don’t show it.

Reply:
okay!